docs/rel_0.5.0.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Version 0.5.0
+keywords: apache, hudi, release, 0.5.0, data lake, upsert,
+sidebar: mydoc_sidebar
+permalink: rel_0.5.0.html
```

Comment:
I think its better to have the url be just `/releases.html` and then have 0.5.0 as a section?

Reply:
Addressed